pil-analyzer/src/statement_processor.rs

```diff
@@ -144,19 +144,13 @@ where
                     Some(definition),
                 )
             }
-            PilStatement::ConstantDefinition(source, name, value) => {
-                // Check it is a constant.
-                if let Err(err) = self.evaluate_expression_to_fe(value.clone()) {
```

Comment:
why is this not needed anymore?

Reply:
This is used to check if an expression evaluates to a single number that is in the range of the field. The actual number is ignored, though. The evaluation at this point, though, is type-blind: It assumes all numbers are integers. It is much better to just let the type checker check that the expression properly evaluates to `fe`.